bedrock/base/templates/includes/canonical-url.html

```diff
@@ -12,6 +12,33 @@
         <link rel="alternate" hreflang="en" href="{{ settings.CANONICAL_URL + '/' + code + loop_canonical_path }}" title="English">
         <link rel="alternate" hreflang="en-US" href="{{ settings.CANONICAL_URL + '/' + code + loop_canonical_path }}" title="English (USA)">
         <link rel="alternate" hreflang="en-CA" href="{{ settings.CANONICAL_URL + '/' + code + loop_canonical_path }}" title="English (Canada)">
+        {% elif code == 'es-ES' -%}
```

Comment:
Only for `es-ES`? We support several `es-XX` locales. We should include `es` if any of them are active right? Perhaps we could improve the code list in the context to include all 2 letter locales?

Reply:
Right, but (iiuc) the ask here is to provide Google with a hint to the default language (not locale) for Spanish, which should probably always be `es-ES` (same as we have already done for `en-US`). We're not doing this for every two letter locale i.e. `zh-CN` and `zh-CW`. Is that right, @rraue?

Reply:
Right, but what I’m saying is that a page might have only `es-AR` enabled, and then `es` wouldn’t show up. It might be rare enough to not matter. But we could create a more official list of default locales for ones where we have more than one regional locale. This is probabally good enough for now, but we should keep in mind that it could result in these languages beeing missing if these specific regions aren’t activated.

Reply:
Hmm, yeah - I don't know how likely that is - but I guess it could happen. If `es-ES` isn't enabled for a page however, would it be wise to still point Googlt to it? Or would it be better to leave it out?

Reply:
I guess the question is really whether we want only a specific locale to be the default for a two letter code and not be there if that locale isn’t active, or to try to choose the best for a default from the list that is active?

Reply:
Right, I guess that's a question for @rraue to answer.

Reply:
You are both right. I assumed es-ES is always translated, if one of the other spanish locales is. That might be wrong and then we would create more missing confirmation links like in https://bugzilla.mozilla.org/show_bug.cgi?id=1442339. 

Of course es can be only present if es-ES exists. I took es-ES as the default for es because it has more pages translated then the other locales, but it might get us some issues. Not sure how to check that. 

The same of course is valid for the other added languages, the locale pointing to needs to exists so we link the language.

Reply:
So what is the best recommended course of action here? Should we proceed as-is, or should we default to e.g. `es` being `es-ES` if available, else falling back to another `es` locale?

Reply:
So for now, let us just include es when es-ES exists. Same logic for the other added languages. I need to understand our workflow for translations better to recomment a broader solution. This will at least solve the problem for our most important pages, as they exist in es-ES and we do not add any new issue.

So at least any spanish version for the firefox download ranks in spanish speaking countries not having a locale. We rank for "descargar firefox" with the english version of /new/ on no. 1 in Colombia, Peru, Ecuador, Venezuela and so on and this would be fixed.

Reply:
We could go all out and check the lang files and see if e.g. es-MX is ever activated without es-ES, but my guess is that you’re probably right that that’s rare, and it will certainly not be the case on the really popular pages. I think we can go ahead with this and then improve later if this really does come up.

Reply:
Thanks both, sgtm 🌮